include/tscore/CryptoHash.h

```diff
@@ -107,9 +109,20 @@ union CryptoHash {
 
   /// Fast conversion to hex in fixed sized string.
   char *toHexStr(char buffer[(CRYPTO_HASH_SIZE * 2) + 1]) const;
-};
 
-extern CryptoHash const CRYPTO_HASH_ZERO;
+  bool
+  is_zero() const
+  {
+    const uint8_t z64[sizeof(u64)] = {0};
```

Comment:
nitpicking: would you make this `constexpr`? because `sizeof(u64)` is defined in the compile time.

Reply:
Done!